specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -1188,11 +1188,17 @@ model AcsMessageReceivedEventData extends AcsMessageEventData {
   /** The message content */
   content?: string;
 
+  /** WhatsApp message type */
```

Comment:
to confirm all these types are optional?

Reply:
Posted message to CPM team just to confirm. I am sure they are optional. Will send an update once I receive a reply back. Thanks Libba.

Reply:
Hello Libba, 
Yes, they all are optional. Please let me know if you have any questions. Thanks.